asset-services/twilio-verify/src/client.rs

```diff
@@ -29,6 +29,31 @@ impl VerifyClient {
         ))
     }
 
+    /// Fetch the verification service configuration for the current service.
+    pub async fn fetch_current_service(
```

Comment:
Not a 100% on the twillio api conventions, but it might be a little clearer to call this `fetch_current_service_config`. (same for the `fetch_service` below).

Unfortunately "Service" has a lot of meanings, so a little bit more clarity might help.

Reply:
Yeah, the Twilio API is remarkably Not Well Named here. 😄

The Twilio Verify API docs use "Service" (or "Verification Service") consistently, so I erred towards matching that terminology and convention here: I figure it's better to have to learn their resource naming convention once, rather than having to deal with both their naming convention and a slightly different one for our client.